docs/utilization.md

```diff
@@ -2,157 +2,201 @@
-An employee is considered `Non billable` if their `TARGET BILLABLE HOURS` over a time period is 0. Utilization is not calculated for these individuals because
-we're currently unable to <a href="https://en.wikipedia.org/wiki/Division_by_zero">divide by zero</a>.
+**Note:** Allocation percentages technically refer to an employee's availability on a project rather than a specific number of hours worked. But we convert this percentage to an _expected_ hourly value to enable utilization rate calculations across durations (e.g., the past month) and groups (e.g., all of `Project ABC`) that may include varying combinations of hourly, weekly, and excluded hours projects.
 
-All `BILLABLE HOURS` tocked by `Non billable` employees are included in aggregations for their respective business unit and organization.
+### Target billable hours
+A user's expected number of hours tocked to billable projects over a period of time. Determined by their [billing expectation](#billing-expectation) over that period and the total number of hours in a week less any hours which were not worked like `Out of office`.
+
+```
+TARGET_BILLABLE_HOURS = (HOURS_IN_WEEK - HOURS_NOT_WORKED) * BILLING_EXPECTATION
+```
+
+### Billing expectation
+The percent of [hours in a week](#hours-in-a-week) which are expected to be billable for a given reporting period. Set per user, per timecard, and will vary as an individual's role, position, or organization change. See [the handbook](https://handbook.tts.gsa.gov/tock/#weekly-billable-hour-expectations) for additional details on billing expectations.
 
 **Default: 80%**
 
+### Hours in a week
+The number of hours in a standard week, i.e., `40`. This value is static.
+
+### Non-billable employee
+An employee is considered non-billable if their [target billable hours](#target-billable-hours) over a time period is 0. Utilization is not calculated for these individuals because we're currently unable to <a href="https://en.wikipedia.org/wiki/Division_by_zero">divide by zero</a>.
+
+All billable hours and allocation tocked by non-billable employees are included in aggregations for their respective business unit and organization.
 ## Example
 
-Stacey is an active and billable 18F employee, here's a look at their recent tock activity! As an individual contributor, Stacey has a `BILLING_EXPECTATION` of 80%.
+Stacey is an active and billable 18F employee, here's a look at their recent tock activity! As an individual contributor, Stacey has a billing expectation of 80%.
 
 Project | Week 1 | Week 2 | Week 3 | Week 4
 --------|--------|--------|--------|--------
 Out of Office| 0.0 | 8.0 | 0.0 | 0.0
-Billable | 24.0 | 28 | 32.0 | 32.0
-Non-Billable | 16.0 | 4.0 | 8.0 | 8.0
+Hourly Billable | 24.0 | 30.0 | 0.0 | 32.0
+Weekly Billable | 25% | 0% | 100% | 0%
+Hourly Non-Billable | 8.0 | 2.0 | 8.0 | 8.0
 
 ### Week 1
-Stacey's `utilization` for Week 1 is 75%.
-Stacey tocked 24 of an expected 32 billable hours.
+Stacey's `utilization` for Week 1 is 100%.
+
```

Comment:
> 8 billable hours to a weekly project

Justo be super clear, is 8 the result of 32 hors per 25%, can we add that calculation as a parenthesis here?

Reply:
Added the parenthetical to the line below